nix/rollup-bridge-contracts.nix

```diff
@@ -45,9 +48,38 @@ stdenv.mkDerivation rec {
     npx hardhat clean && npx hardhat compile
   '';
 
+  doCheck = enableTesting;
+  checkPhase = ''
+    export WALLET_ADDRESS=0xc8d5559BA22d11B0845215a781ff4bF3CCa0EF89
```

Comment:
not sure about hardcoded these values here

Reply:
Corrected